docs/content/connectors/oracle-cdc.md

```diff
@@ -205,8 +205,8 @@ Flink SQL> CREATE TABLE products (
      'port' = '1521',
      'username' = 'flinkuser',
      'password' = 'flinkpw',
-     'database-name' = 'XE',
-     'schema-name' = 'inventory',
+     'database-name' = 'XE',  -- XE is oracle SID
+     'schema-name' = 'inventory',  -- inventory is oracle user
```

Comment:
dabase-name could accept non SID value, schema=name caould accept non user value

Reply:
I have some confusion.

I found the following answer in a link.

https://dba.stackexchange.com/questions/37012/difference-between-database-vs-user-vs-schema

Oracle's definitions of user and schema seem to be the same. Of course, this is limited to Oracle, which is not the case for other databases.

Reply:
ping @luoyuxia could you also help have a check ?

Reply:
@rookiegao Hi, so is it possible that user 'SCOTT‘ create a table `inventory.t1'`? Then it seems the `inventory` will be the `schema-name`.

Reply:
I see. I'll cancel the modification